### ISSUE TITLE
PolyData readers not producing vertex cells

### ISSUE THREAD

**Issue by banesullivan** [pyvista/pyvista#227]
PyVista reading routines that output  `PolyData` are not producing vertex cells

After running snippet in: https://github.com/pyvista/pyvista/issues/226#issuecomment-494032571

The following results (note the produced PolyData from the file has zero cells):

```python
In [54]: pyvista.wrap(sphere.GetOutput())
Out[54]: 
PolyData (0x7f77bcfa3768)
  N Cells:	96
  N Points:	50
  X Bounds:	-4.875e-01, 4.875e-01
  Y Bounds:	-4.875e-01, 4.875e-01
  Z Bounds:	-5.000e-01, 5.000e-01
  N Scalars:	1

In [55]: pyvista.PolyData('test.vtk')
ERROR:root:,5
Out[55]: 
PolyData (0x7f77bcfa3a08)
  N Cells:	0
  N Points:	50
  X Bounds:	-3.109e-02, 4.842e+30
  Y Bounds:	-5.181e+04, 6.380e+28
  Z Bounds:	-5.181e+04, 7.923e+34
  N Scalars:	0
```

_Originally posted by @supersubscript in https://github.com/pyvista/pyvista/issues/226#issuecomment-494048734_

**Comment by banesullivan** [comment]
Important to note that `pyvista.read()` has the same behavior with STL files and other PolyData formats.

I quite commonly do this to mitigate the issue:

```py
import pyvista as pv
bad = pv.read('mesh.stl')
poly = pv.PolyData(bad.points)

**Comment by banesullivan** [comment]
Resolved in 7b6931e with a simple check to see if there are points but now cells upon object instantiation.

To test:

```py
>>> import pyvista as pv
>>> import numpy as np
>>> pv.PolyData(np.random.rand(100, 3)).save('foo.ply')
>>> pv.read('foo.ply').n_cells == 100
True
```
